app/bundles/LeadBundle/Entity/LeadRepository.php

```diff
@@ -578,6 +578,34 @@ public function getEntities($args = array())
             ) : $results;
     }
 
+
+    public function getEntitiesLight($args = array()){
+    	$limit = 20000; //lines
+    	$results = array();
+    	$offset = 0;
+    	$result = $this->requestEntitiesLight($args, $offset, $limit);
+    	$results = array_merge($results, $result);
+    	while (count($result) === $limit){
+    		$offset += $limit;
+    		$result = $this->requestEntitiesLight($args, $offset+1, $limit);
+    		$results = array_merge($results, $result);
+    	}
+    	return $results;
+    }
```

Comment:
What is the purpose of this method? To avoid a memory limit in the SQL server? I think it will fail on the php memory limit first in most cases.